eng/common/cross/build-rootfs.sh

```diff
@@ -5,7 +5,7 @@ set -e
 usage()
```

Comment:
the arcade update PR with these changes was just merged, you can remove the changes here

Reply:
Merged main into my branch and they were squashed. :)